Presentation.Web/app/shared/helpText/helpText.directive.ts

```diff
@@ -10,25 +10,47 @@
                 noButtonLayout: "@"
             },
             controller: [
-                "$scope", "$uibModal", "helpTextService", ($scope, $uibModal, helpTextService : Kitos.Services.IHelpTextService) => {
+                "$scope", "$uibModal", "helpTextService", "notify", ($scope, $uibModal, helpTextService: Kitos.Services.IHelpTextService, notify) => {
                     var parent = $scope;
 
                     $scope.showHelpTextModal = () => {
                         $uibModal.open({
                             windowClass: "modal fade in",
                             templateUrl: "app/shared/helpText/helpTextModal.view.html",
-                            controller: ["$scope", ($scope) => {
+                            controller: ["$scope", "$uibModalInstance", ($scope, $uibModalInstance) => {
+                                const helpTextKey = parent.key;
 
-                                helpTextService.loadHelpText(parent.key)
+                                helpTextService.loadHelpText(helpTextKey)
                                     .then(helpText => {
                                         if (helpText != null) {
                                             $scope.title = helpText.title;
                                             $scope.description = helpText.htmlText;
                                         } else {
                                             $scope.title = parent.defaultTitle;
                                             $scope.description = "Ingen hjælpetekst defineret.";
+
+                                            helpTextService.createHelpText(helpTextKey, $scope.title);
```

Comment:
it should be created if the Edit button is clicked and that should only be visible if the user can actually edit help texts